pkg/operator/ceph/object/user/controller.go

```diff
@@ -168,13 +172,27 @@ func (r *ReconcileObjectStoreUser) reconcile(request reconcile.Request) (reconci
 		}
 		return reconcileResponse, nil
 	}
+	r.cephClusterSpec = &cephCluster.Spec
 
 	// Set a finalizer so we can do cleanup before the object goes away
 	err = opcontroller.AddFinalizerIfNotPresent(r.client, cephObjectStoreUser)
 	if err != nil {
 		return reconcile.Result{}, errors.Wrap(err, "failed to add finalizer")
 	}
 
+	// Populate clusterInfo
+	// Always populate it during each reconcile
```

Comment:
We have to reload it every time in case it changes?

Reply:
Yes, I'd say better safe than sorry, e,g, if the mon changes.